src/main/java/org/broadinstitute/hellbender/engine/datasources/ReferenceWindowFunctions.java

```diff
@@ -14,7 +14,23 @@
     /**
      * A function for requesting only reference bases that directly overlap each read
      */
-    public static final SerializableFunction<GATKRead, SimpleInterval> IDENTITY_FUNCTION = read -> new SimpleInterval(read);
+    public static final SerializableFunction<GATKRead, SimpleInterval> IDENTITY_FUNCTION = new IdentityFunction();
+
+    /**
+     * A function that returns all bases overlapped by the read. To maintain the ability to serialize the reference,
+     * this needs to be a class rather than a simple lambda function because of errors in serializing lambda functions
```

Comment:
OK - I was thinking it'd be nice to mimic the style of the other function (FixedWindowFunction) in the containing class. Changed to an anonymous class, unit test still passes.